backend/src/main/java/com/sim_backend/websockets/enums/ErrorCode.java

```diff
@@ -26,7 +26,7 @@ public enum ErrorCode {
    * Payload for Action is syntactically correct but at least one of the fields violates occurrence
    * constraints.
    */
-  OccurenceConstraintViolation,
```

Comment:
The Spec has it as OccurenceConstraintViolation even though it's not correct. The code that uses this just toString this enum so make sure that code is fixed or keep it as is.

Reply:
Reverted